public/api/v1/getuserid.php

```diff
@@ -16,9 +16,17 @@
 
 =========================================================================*/
 
+$noforcelogin = 1;
```

Comment:
Is this required to before all the includes/requires, or just one (or none, even)? I would prefer it to be just before the include/require where it is used.